assets/messages.yml

```diff
@@ -87,24 +69,37 @@ variables:
   awaiting_response:
     - project:
         - mc
-        - mcd
         - mcpe
         - mcl
-        - mclg
         - bds
         - realms
         - web
       value: |-
         ~This issue is being temporarily resolved as {color:#d04437}*Awaiting Response*{color}. Once the requested information has been delivered, the report will be reopened automatically.~
 
+  wai_links:
+    - project: bds
+      value: |-
+        [Full Version History|https://minecraft.wiki/w/Bedrock_Dedicated_Server#History] -- [The official Minecraft feedback site|https://feedback.minecraft.net]
```

Comment:
We always say wiki is not a WAI source. Should we be linking to it when leaving messsages about WAI?

Reply:
Oh I see where it is placed now. We do have it in the current messages.

Wondering still if it's good to link to those in a WAI message though

Reply:
I wouldn't want to deviate from bugnet, but I'm not against this per se (see #87 also).

Reply:
Oh does even bugnet put this? I guess we keep it then to maintain parity and look at 87 down the line